src/Illuminate/Foundation/helpers.php

```diff
@@ -590,13 +590,13 @@ function env($key, $default = null)
 			case '(false)':
 				return false;
 
-			case 'null':
-			case '(null)':
-				return null;
-
 			case 'empty':
 			case '(empty)':
 				return '';
+
+			case 'null':
+			case '(null)':
+				return;
```

Comment:
I agree with the others, but I disagree with this one. I think this one should be an explicit `null` return.